include/seqan3/alphabet/nucleotide/dna3bs.hpp

```diff
@@ -0,0 +1,189 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
+ * \brief Provides seqan3::dna3bs, container aliases and string literals.
+ */
+
+#pragma once
+
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/nucleotide_base.hpp>
+#include <seqan3/core/char_operations/transform.hpp>
+
+// ------------------------------------------------------------------
+// dna3bs
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
+/*!\brief The three letter reduced DNA alphabet for bisulfite sequencing mode (A,G,T(=C)).
+ * \ingroup nucleotide
+ * \implements seqan3::NucleotideAlphabet
+ * \implements seqan3::WritableAlphabet
+ * \if DEV \implements seqan3::detail::WritableConstexprAlphabet \endif
+ * \implements seqan3::TriviallyCopyable
+ * \implements seqan3::StandardLayout
+ * \implements std::Regular
+ *
+ * \details
+ * Note that you can assign 'U' as a character to dna3bs and it will silently
+ * be converted to 'T'.
```

Comment:
I would remove this sentence here as it is not specific to the dna3bs alphabet, but looks like it is.